Dockerfile_ssportal_migration

```diff
@@ -0,0 +1,21 @@
+FROM node:18-alpine
+ENV NODE_ENV production
+WORKDIR /usr/src/app
+
+# Copy the relevant files for migration neeeds
+COPY *.js *.ts *json .npmrc ./
+COPY src/api/envars.ts ./src/api/
+COPY src/database/migrations/* ./src/database/migrations/
+
+# The image set ENV production, but we need the dev deps to build, so install with prod false
+RUN npm ci --production=false
+
+# Delete the status completion file if present
+RUN rm -f /var/run/migration-status/completed
```

Comment:
just curious, why does this need to be deleted, is this file blocking the migration?